### DISCUSSION TITLE
Self hosted certificate authority not supported (SSL errors)

### DISCUSSION THREAD

**Discussion by Blyyyn** [requarks/wiki#3527] [Error / Bug Report]
Greetings,

```
OS : CentOS Linux release 7.8.2003
Docker : docker-ce-19.03.11-3.el7.x86_64
PGSQL Image : postgres:12-alpine
WikiJS Image : requarks/wiki:2
```

I already reported my issue on another thread, which was already closed and you may not noticed my comment.

https://github.com/Requarks/wiki/issues/1120#issuecomment-633218304

I would like to know if you can implement some feature on your application, allowing us to integrate any trusted certificate authority we have under control. The solution proposed on the above post **is more a workaround than a solution** (to me) : i don't want do disable the certificate authority verification, i just want to trust the ones i manage. Is there a solution i can use ? Do you plan ton do something about it ?

Maybe there is a way to add trusted certificates when building the image (i'm usine docker-compose right now).

Thanks !

**Comment by Smankusors** [comment]
IMO, I think this should be feature request,,, 🤔

**Comment by Blyyyn** [comment]
I dont really know, yeah it could.

According to me this could also be a security issue (the only way to accept self signed certificate authority is ... to disable certificate authority checking...). Depends on ur security sensitivity i guess :)

**Comment by Blyyyn** [comment]
Any news ?
